core/src/tvu.rs

```diff
@@ -357,6 +359,8 @@ impl Tvu {
             )
         });
 
+        let ledger_metric_report_service = LedgerMetricReportService::new(blockstore);
```

Comment:
Normally this `blockstore` should be `blockstore.clone()`, but rust check complains that this is the very last use of `blockstore` and thus `clone()` here is unnecessary.